src/trackers/TVC.py

```
@@ -362,148 +532,272 @@ async def search_existing(self, meta, disctype):
+            screens_count = int(meta.get('screens', 0) or 0)
+            if image_list and screens_count >= self.config['TRACKERS'][self.tracker].get('image_count', 2):
+                desc += "\n\n[b]Screenshots[/b]\n\n"
+                for each in image_list[:self.config['TRACKERS'][self.tracker]['image_count']]:
+                    web_url = each['web_url']
+                    img_url = each['img_url']
+                    desc += f"[url={web_url}][img=350]{img_url}[/img][/url]"
+            desc += "[/center]\n\n"
+
+        # Movie / fallback overview
+        else:
+            # Fallback path: for non‑movie categories with only a generic overview available.
```

Ruff (0.14.8)

Comment contains ambiguous `‑` (NON-BREAKING HYPHEN). Did you mean `-` (HYPHEN-MINUS)?

(RUF003)